source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -164,9 +274,83 @@ provision the following networking API entities, represented in yaml:
       - subnet_id: "{{ ipv6_subnet.id }}"
 ~~~~~
 
+### Configuring stateless DHCPv6 on the subnet
+
+To configure stateless Dhcpv6 on the subnet, the user should provision
+the following yaml based ansible playbook snippet:
+
+~~~~~
+
+- name: create a network
+  os_network:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: netv6
+  register: ipv6_network
+
+- name: attach a subnet to the logical network
+  os_subnet:
+    cloud: "{{ cloud_name }}"
+    state: present
+    network_name: netv6
+    name: subnet_ipv6
+    ip_version: 6
+    cidr: bef0:1234:a890:5678::/64
+    gateway_ip: bef0:1234:a890:5678::1
+    ipv6_address_mode: dhcpv6_stateless
+  register: dhcpv6_subnet
+
+- name: create a logical switch port
+  os_port:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: lport1
+    network: "{{ ipv6_network.id }}"
+    fixed_ips:
+      - subnet_id: "{{ dhcpv6_subnet.id }}"
+
+- name: create a router
+  os_router:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: router0
+    interfaces:
+      - "{{ dhcpv6_subnet.id }}"
+
+~~~~~
+
+## UI considerations
+
+Having dynamic address assignment in EL8 requires stateless dhcpv6 subnet
+configuration, which in turn requires an OVN logical router, in order for
+the RA generation logical flows to be installed.
+
```

Comment:
Something is not yet clear to me. 
If i want to use this RFE in oVirt UI with rhel8 guest, what i(user/admin) need to do?
1. Create network with stateless dhcpv6 subnet
2. Create router? here it is not clear to me, does user need to create a router for a single IPv6 subnet using dhcpv6 stateless also? if so, it means that in rhel8 guest, everything must go via router, even single subnet?
3. Why logical switch port is required? 
4. Can you please provide an example of how "To configure stateless Dhcpv6 on the subnet" via REST API?
this will be very much useful.


```

Reply:
You're making a ton of question in a single comment, and this is gonna be hard to cover, but I'll try.

1. Yes, you create a network that is attached to a stateless dhcpv6 subnet.
2. Yes, you have to create a router. That's not possible through the UI. And yes, even a single subnet requires a router. Why ? Read the section [0]; TL;DR => network prefixes are configured from RA messages, which are sent by routers.
3. Because a logical switch port is what connects a VM to an OVN logical switch - it represents your UI created VMs
4. I'm providing it via ansible. Mapping it from ansible to rest is straightforward. 

[0] - https://github.com/oVirt/ovirt-site/blob/5533edd1ce95cb380a8109b24a9e5051306e71c4/source/develop/release-management/features/network/ipv6_subnets.html.md#stateless-dhcpv6--slaac-motivation

Reply:
4. It is not straight forward to all. 
Requested 4 examples of rest api body requests in previous comment

Reply:
REST API examples provided, as requested. 

Let me know if they're OK.

Reply:
OK, thanks!